frontend/src/admin/common/forms/InputField.js

```diff
@@ -10,14 +12,14 @@ const InputField = ({ input, label, placeholder, type, optional, id, meta: { tou
 
   return (
     <div>
-      <label
+      <Label
         htmlFor={input.name}
         className={classNames()}
```

Comment:
There are some conditional classNames in here which I suspect now don't do anything? Might need removing/replacing?